pallets/teerex/src/lib.rs

```diff
@@ -637,6 +645,20 @@ pub mod pallet {
 			// Do not pay a fee
 			Ok(Pays::No.into())
 		}
+
+		/// This extrinsic is used to set ScheduleEnclave storage item
+		/// This storage item is used to perform feature control during register_enclave
+		#[pallet::call_index(31)]
+		#[pallet::weight((195_000_000, DispatchClass::Normal, Pays::No))]
+		pub fn set_schedule_enclave(
+			origin: OriginFor<T>,
+			schedule_enclave: bool,
+		) -> DispatchResultWithPostInfo {
+			ensure_root(origin)?;
```

Comment:
I think the admin account origin check is better

Reply:
Updated to use `TeerexAdmin`